core/domain/user_services_test.py

```diff
@@ -485,11 +485,16 @@ class UserDashboardStatsTests(test_utils.GenericTestBase):
 
     USER_SESSION_ID = 'session1'
 
+    CURRENT_DATE_AS_STRING = utils.get_current_date_as_string()
+
     def setUp(self):
         super(UserDashboardStatsTests, self).setUp()
         self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
         self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
 
+    def _mock_get_current_date_as_string(self):
```

Comment:
This is good. Technically, you should do this in the other tests as well (for DashboardStatsOneOffJobTests), otherwise you have problems if someone runs the tests around midnight.